packages/pwa-kit-dev/src/ssr/server/build-dev-server.test.js

```diff
@@ -556,7 +556,7 @@ describe('DevServer persistent caching support', () => {
         route = null
     })
 
-    test('Caching of compressed responses', () => {
+    test('No caching of compressed responses', () => {
```

Comment:
Are we sure we want to test the non-existence of a deprecated feature? If "yes" is the answer, how do we define a time when we would deem it safe (in the future) to delete this test?

Reply:
> These changes are to remove the PersistentCache functionality in the PWA Kit before removing the entire PersistentCache feature in a future release.

Ok, so we're removing from PWA Kit but leaving in MRT temporarily? I thought it was already verified that no existing customer implementation is using the feature (per previous conversation)?

Reply:
I left the tests in there currently because I didn't want the test coverage to drop and then have to make it up in other places, but I am fine with removing all the tests and bringing down the coverage level.

I'll update the description to be more clear but we have already removed it from the managed runtime. The managed runtime team discussed removing the functionality of it from the PWA Kit first so not to make a breaking change and then in a future V3 release would remove the code entirely.

Reply:
One other thing I forgot to mention is that I left the tests in there as well because they cover pieces that surround the PersistentCache feature there are unchanged `sendCachedResponse`, `cacheResponseWhenDone`, `getResponseFromCache`. Those functions are unchanged (but they won't do anything since the cache no longer works).